src/middlewares.js

```diff
@@ -110,6 +110,11 @@ export function handleParseHeaders(req, res, next) {
   req.config = new Config(info.appId, mount);
   req.info = info;
 
+  const ip = req.ip;
```

Comment:
hum, how does it work behind a LB? req.ip may be the LB's IP and not the original client IP right?